Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -7,12 +7,14 @@ import CardBanner from './CardBanner';
 
 const CardContainer = () => {
 	return (
-		<div id="cardContainer">
-			<div id="smallerContainer">
-				<CardBanner />
-				<CardContent />
+		<a href="https://www.reactjs.org">
+			<div id="cardContainer">
```

Comment:
Using `.classes` over `#ids` would work just as well here. Id's are _very_ specific. If that was your intention then great, but just know using classes here would be fine.